docs/getting_started/usage.md

```diff
@@ -0,0 +1,75 @@
+# Usage overview
+
+The Usage page provides users with a comprehensive overview of their data consumption and performance metrics associated with their communities. This feature is essential for understanding resource allocation and ensuring compliance with the fair use policy. Users can access detailed analytics regarding their analyzed bytes by community, as well as all executed playbook actions, facilitating efficient monitoring of usage and prevention of service degradation.
+
+## How it works
+
+Upon accessing the Usage page, you are presented with a dashboard that displays their data usage for the current month. The dashboard consists of key sections:
+
+- **Analyzed Bytes**: Visualizes the total data analyzed by the community and indicates whether the usage is within acceptable limits.
+- **Executed Playbook Actions**: Summarizes automation actions taken based on user-defined playbooks. 
+
+As a member of a workspace, you can quickly review consumption metrics across all your communities. The summary table includes:
+
+- **Community name**
+- **Number of subscribed assets**: Total subscribed assets in each community (based on what has been defined in your contract with Sekoia). 
+- **Type of subscription**: The subscription plan type.
+- **Analyzed bytes**: The amount of data analyzed, applicable to your plan.
+
+You can also have a look at your previous consumption by clicking on the date selector in the upper right of the screen. 
+
+## Sections
+
+### Analyzed bytes
+
+In the Analyzed bytes section, users can view a detailed breakdown of their monthly data usage in relation to the Fair Use Policy.
+
+- **Current usage**: Displays the total bytes analyzed (e.g., 100 GB) and highlights any excess usage against the predefined fair use limit (e.g., 210 GB).
+- **Fair use indicator**: A visual marker shows whether current consumption exceeds the fair use threshold.
+
+### Fair use policy
```

Comment:
Not sure we should document our usage policies in this document. It should be in the contract.

Reply:
@gbossert you recommend removing the whole section? I think we need to explain what we mean by fair use and how it is calculated